RomanGupalo/golang_bootcamp/server/db/connection/adapter.go

```diff
@@ -0,0 +1,23 @@
+package connection
+
+import (
+	"github.com/upper/db/v4"
+	"github.com/upper/db/v4/adapter/mysql"
+)
+
+// created for testing
+type i_adapter interface {
+	Open(connURL db.ConnectionURL) (db.Session, error)
+}
+
+type mysql_adapter struct{}
```

Comment:
No snake_case is used in Golang, only CamelCase please:
```suggestion
type mysqlAdapter struct{}
```